packages/camera/camera_platform_interface/lib/src/types/video_capture_options.dart

```diff
@@ -0,0 +1,52 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+
+import 'camera_image_data.dart';
+
+/// Options wrapper for [CameraPlatform.startVideoCapturing] parameters.
+@immutable
+class VideoCaptureOptions {
+  /// Constructs a new instance.
+  const VideoCaptureOptions(
+    this.cameraId, {
+    this.maxDuration,
+    this.streamCallback,
+    this.streamOptions,
+  });
+
+  /// The ID of the camera to use for capturing.
+  final int cameraId;
+
+  /// The maximum time to perform capturing for.
```

Comment:
Separate the first sentence into it's own paragraph: https://dart.dev/guides/language/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph.

Same for the fields below.

Reply:
Done.